### ISSUE TITLE
Tried to install in a jupyter docker container, failed

### ISSUE THREAD

**Issue by dionjwa** [Autodesk/molecular-design-toolkit#107]
```
15:45 $ docker run -ti jupyter/jupyterhub /bin/bash
Unable to find image 'jupyter/jupyterhub:latest' locally
latest: Pulling from jupyter/jupyterhub
efd26ecc9548: Pull complete
a3ed95caeb02: Pull complete
298ffe4c3e52: Pull complete
758b472747c8: Pull complete
8b9809a68afc: Pull complete
93b253b5483d: Pull complete
ef8136abb53c: Pull complete
Digest: sha256:8f8cd2b62942b29e84bb99401ec9819d489b0cf0ebece99a42ba05946feeb72f
Status: Downloaded newer image for jupyter/jupyterhub:latest
root@766a9d01a6b4:/srv/jupyterhub# pip install moldesign
Collecting moldesign
  Downloading moldesign-0.7.3.tar.gz (20.0MB)
    100% |████████████████████████████████| 20.0MB 69kB/s
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-jbo8ia1m/moldesign/setup.py", line 70
        print 'Thank you for installing the Molecular Design Toolkit!!!'
                                                                       ^
    SyntaxError: Missing parentheses in call to 'print'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-jbo8ia1m/moldesign/
You are using pip version 8.1.1, however version 8.1.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
111

**Comment by dionjwa** [comment]
It would be really great if there was a Dockerfile in the root directory that had the correct versions of everything to run and test, independent of the host setup.

**Comment by dionjwa** [comment]
Also tried to install and run on my machine. It installed ok with `sudo`, but could not run:
```19:26 $ python -m moldesign intro
/usr/bin/python: No module named moldesign
✘-1 ~/autodesk/cloud-compute-cannon [stream_std_out_err_wip L|✚ 3⚑ 2]
19:26 $ which pip
/usr/local/bin/pip
✔ ~/autodesk/cloud-compute-cannon [stream_std_out_err_wip L|✚ 3⚑ 2]
19:27 $ which python
/usr/bin/python```

**Comment by avirshup** [comment]
Thanks @dionjwa . 

 1. It doesn't install in the docker image because it's defaulting to Python 3, and MDT is still python 2 only. However, it should have printed out an error message stating that explicitly instead of dying with a syntax error - that's a bug.
 1. Agreed, we need a root Dockerfile. Will add it.
 1. Local installation doesn't work: :headdesk: Python + MacOS = sadness. Probably try switching to the Homebrew python install, or, per @hainm in #32, try `pip install --user`.

**Comment by avirshup** [comment]
FYI @dionjwa - actually, the easiest way to deploy, as long as you're not doing development, is to pull the official docker image:

`docker run -p 8888:8888 -it autodesk/moldesign:moldesign_notebook-0.7.3`

**Comment by avirshup** [comment]
Fixed with #108
